homeassistant/components/media_player/services.yaml

```diff
@@ -184,3 +184,30 @@ repeat_set:
             - "off"
             - "all"
             - "one"
+
+join:
+  description:
+    Group players together. Only works on platforms with support for player
+    groups.
+  fields:
+    entity_id:
```

Comment:
Please use `target:` instead. See other service descriptions for examples.

Reply:
OK, fixed.